config/core/test-configs.yaml

```diff
@@ -1640,6 +1640,14 @@ device_types:
             - 'tinyconfig'
             - 'kvm_guest.config'
 
+  xilinx-zcu102:
+    mach: zynq
+    class: arm64-dtb
+    boot_method: uboot
+    filters:
+      - blocklist: *allmodconfig_filter
+      - blocklist: {defconfig: ['multi_v7_defconfig+linaro-base+distribution']}
```

Comment:
Where is `+linaro-base+distribution` coming from? As far as I know there aren't any fragment called like that in build-configs.yaml.

Reply:
Ah fair enough, I guess these fragments were added there as well by mistake.  Could you please drop it from `xilinx-zcu102` and maybe also add a patch to drop it from `zynq-zc702`?